thinc/layers/with_signpost_interval.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional, Callable, Any, Tuple, TypeVar
+
+from ..compat import has_os_signpost, os_signpost
+from ..model import Model
+
+
+_ModelT = TypeVar("_ModelT", bound=Model)
+
+
+def with_signpost_interval(
+    layer: _ModelT,
+    signposter: "os_signpost.Signposter",
+    name: Optional[str] = None,
+) -> _ModelT:
+    """Wraps any layer and marks the forward and backprop phases using
```

Comment:
```suggestion
    """Wraps any layer and marks the init, forward, and backprop phases using
```

Reply:
`init` added here and `api-layers.md` without Oxford commas.